src/backup

```diff
@@ -11,13 +11,14 @@ if [ -z "$MYSQL_ENV_MYSQL_HOST" ]; then echo "Error: MYSQL_ENV_MYSQL_HOST not se
 if [ -z "$MYSQL_ENV_MYSQL_USER" ]; then echo "Error: MYSQL_ENV_MYSQL_USER not set"; echo "Finished: FAILURE"; exit 1; fi
 if [ -z "$MYSQL_ENV_MYSQL_DATABASE" ]; then echo "Error: MYSQL_ENV_MYSQL_DATABASE not set"; echo "Finished: FAILURE"; exit 1; fi
 if [ -z "$MYSQL_ENV_MYSQL_PASSWORD" ]; then echo "Error: MYSQL_ENV_MYSQL_PASSWORD not set"; echo "Finished: FAILURE"; exit 1; fi
+if [ -z "$BACKUP_NAME" ]; then export BACKUP_NAME=backup; fi
```

Comment:
You should check here, that the variable is acutally set, like the other variables before. $BACKUP_NAME **cannot** be unset here, because it was either set by an explicit env variable, or by the default in the Dockerfile. It is definitely an error, if it has no value here.